elated-egrets/bangazon-internal-site-2.0
# Rm complete order

## DESCRIPTION
added ability for the user to complete an order
closes #21 

## How to pull down
Commands

```
git fetch --all
git checkout rm-complete-order
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. remake migrations if nessesary
1. go through steps to create necessary data, you will need a user, payment type for that user, and an item to buy
1. add an item to  the cart
1. go to the shopping cart and confirm the item is still there
1. click the order button 
1. select a payment type to use for the order
1. complete the order, viewing the order in your database it should now have a date closed and payment type attached
1. if you go back to the item you purchased it should now be missing one quantity, and viewing your cart should say you have no open orders

## What you changed
created a new template/view for adding a payment type to an order
added logic for completing an order

## What was added to read me

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Have 2 teammates approved the PR?

Then Ship It!
